README.md

```diff
@@ -10,14 +10,37 @@ A description of the features will be added during the development process.
 
 ## Built With
 
-* [The Ruby language](https://www.ruby-lang.org/) - version 2.3.1
-* [Ruby on Rails](https://github.com/rails/rails) - version 5.1
-* [PostgreSQL](https://www.postgresql.org/)
+* The Ruby language -version 2.3.1
+* Ruby on Rails
+* PostgreSQL
 
 ## Deployment
 
 ## Authors
 
-* **Yaroslav Liakh**
-* **Vitaliy Maznyi**
-* **Petr Kujda**
```

Comment:
Please add badge for CI. Here is markdown:
[![Build Status](https://semaphoreci.com/api/v1/ydionis/ruby-hotel/branches/master/shields_badge.svg)](https://semaphoreci.com/ydionis/ruby-hotel)